feder/monitorings/locale/pl/LC_MESSAGES/django.po

```diff
@@ -402,6 +406,11 @@ msgstr ""
 "%(count)d instytucji zostało przypisanych do %(monitoring)s. Wysyłka "
 "wniosków została zaplanowana."
 
+#, fuzzy
```

Comment:
Raczej powinniśmy zatwierdzać tłumaczenia niż stosować `fuzzy`.

Reply:
To moja pierwsza styczność z tłumaczeniami. Myślałam, że to co jest generowane przez `makemessages` jest obowiązkowe. Dzięki za informację. Następnym razem będę zwracała na to uwagę.